src/DataWrangling/Copernicus/Copernicus.jl

```diff
@@ -58,8 +58,8 @@ copernicusmarine_dataset_id(::GLORYSMonthly) = "cmems_mod_glo_phy_my_0.083deg_P1
 struct CMEMSHourlyAnalysis <: CopernicusDataset end
 copernicusmarine_dataset_id(::CMEMSHourlyAnalysis) = "cmems_mod_glo_phy_anfc_0.083deg_PT1H-m"
 
-CopernicusMetadata{D} = Metadata{<:CopernicusDataset, D}
-CopernicusMetadatum = Metadatum{<:CopernicusDataset}
+const CopernicusMetadata{D} = Metadata{<:CopernicusDataset, D}
```

Comment:
Metadata has 3 params, right?

Reply:
Metadatum has one. Let's merge this then I can add the additional `B` parameter to `Metadatum`

Reply:
Yes.

Reply:
if CI passes then let's do it...